test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ]  Kindly make sure the `test.md` file is actually ignored by git and is not present in your GitHub repository in order to meet the project requirements. :wink:

Reply:
Can I use 'git rm test.md' for this action?